sdks/python/apache_beam/runners/direct/transform_evaluator.py

```diff
@@ -491,14 +491,14 @@ class NullReceiver(object):
     def output(self, element):
       pass
 
-  class _InMemoryReceiver(common.Receiver):
```

Comment:
<!--thread_id:cc_151534747_t; commit:7549b5c2ebe2ae47af9066eaf97364a27e828ab5; resolved:1-->
<!--section:context-quote-->
> **chamikaramj** wrote:
> I ran into issues since we are piggybacking StepContext and SDF DoFnInvoker objects in the DoFn. I added an option to disable this pickling only for the SDF case.

<!--section:body-->
Acknowledged.